website/docs/d/iam_permissions_groups.markdown

```diff
@@ -0,0 +1,31 @@
+---
+subcategory : "Account Management"
+---
+
+# ovh_iam_permissions_group (Data Source)
+
+Use this data source to retrieve am IAM permissions group.
+
+## Example Usage
+
+```hcl
+data "ovh_iam_permissions_group" "website" {
+  urn = "urn:v1:eu:permissionsGroup:ovh:controlPanelAccess"
+}
+```
+
+## Argument Reference
+
+* `urn` - URN of the permissions group.
+
+## Attributes Reference
+
+* `name` - Name of the permissions group.
+* `description` - Group description.
+* `allow` - Set of actions allowed by the permissions group.
+* `except` - Set of actions that will be subtracted from the `allow` list.
+* `deny` - Set of actions that will be denied no matter what permissions group exists.
```

Comment:
I find the wording surprising. Especially because a permission group doesn't reference a permission group

What about
```suggestion
* `deny` - Set of actions that will be denied no matter what permissions are defined in the permission group or policies using it.
```